conf/iptables.conf.example

```diff
@@ -34,11 +37,22 @@
 -A input-management-if --protocol tcp --match tcp --dport 9392 --jump ACCEPT
 # Nessus Administration Interface
 -A input-management-if --protocol tcp --match tcp --dport 8834 --jump ACCEPT
+# Galera cluster
```

Comment:
Well we're still not sure about what we'll use. If we drop Galera for sure then it can be removed